dev/com.ibm.websphere.appserver.mpOpentracing-1.0/resources/l10n/com.ibm.websphere.appserver.mpOpentracing-1.0.properties

```diff
@@ -0,0 +1,18 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+# -------------------------------------------------------------------------------------------------
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=The MicroProfile Opentracing specification provides a mechanism to access an opentracing.io Tracer
```

Comment:
I looked at the properties files entry:
The MicroProfile Opentracing specification provides a mechanism to access an opentracing.io Tracer (.)

Just looks like it is missing a period at the end of the sentence.